datasetinsights/stats/visualization/keypoints_plot.py

```diff
@@ -0,0 +1,174 @@
+""" Helper keypoints library to plot keypoint joints and skeletons  with a
```

Comment:
Please run documentation [steps](https://github.com/Unity-Technologies/datasetinsights/blob/master/docs/README.md) to make sure docstrings are generated properly.